sdk/core/core-http/lib/serviceClient.ts

```diff
@@ -432,8 +430,6 @@ function createDefaultRequestPolicyFactories(credentials: ServiceClientCredentia
   if (credentials) {
     if (isRequestPolicyFactory(credentials)) {
       factories.push(credentials);
-    } else {
-      factories.push(signingPolicy(credentials));
```

Comment:
The PR description talks about keeping `signingPolicy` around, but we are removing the support here....

Reply:
Right, `ServiceClient` no longer uses `signingPolicy` but a developer should be free to construct their own `requestPolicyFactories` list that uses it if they wish.  `createDefaultRequestPolicyFactories` only gets invoked in the "default" cause of `ServiceClient` when `ServiceClientOptions.requestPolicyFactories` is not populated.

Previously the `else` case above would only be hit if the `credentials` parameter was a `ServiceClientCredentials` instance, but that type is no longer accepted by the `ServiceClient` interface.

Reply:
Understood